manifests/init.pp

```diff
@@ -101,6 +101,9 @@
 #   An array of PuppetDB servers to collect metrics from. Defaults to `["$settings::certname"]`
 #   A list of PuppetDB servers that will be configured for telegraf to query.
 #
+# @param postgres_host
```

Comment:
Any chance we could have this work with multiple instances instead of just one?

Reply:
This seems possible.  I think I figured out how to add the server tags at least, just need to add:
```
[[inputs.postgresql_extensible]]
<snip>
  outputaddress = "<hostname>"
```
and then iterate over each hostname in the puppet_metrics_dashboard.conf template to create the same set of queries for each server.